### PR TITLE
hotfix: double adding a quote

### PR DESCRIPTION
**Pull request by h4llow3En**

### CODE REVIEW

File: bash.fsr/AddQuoteVC.swift [hunk 1]
```diff
@@ -41,6 +43,7 @@ class AddQuoteVC: UIViewController {
 				self.presentViewController(alert, animated: true, completion: nil)
 			}
 		}
+        self.navigationItem.rightBarButtonItem?.action = Selector("addQuote")
```
Comment (kiliankoe) [inline bash.fsr/AddQuoteVC.swift]: You can also do this in Swift via defer:

``` swift
defer { self.navigationItem.rightBarButtonItem?.action = Selector("addQuote") }
```

Also, the syntax for selectors is changing with the upcoming version of Swift, just FYI: 

``` swift
#selector(addQuote())
```

no more strings \o/

**Comment by kiliankoe** [comment]
I'd actually rather not just set the action to nil here. Even though that should prevent the possible submission of dupes (which also seems to happen inside the bash itself... so it wasn't probably the app's fault), I'd rather change the `rightBarButtonItem` to a UIActivityIndicator instead.

**Comment by h4llow3En** [comment]
It is changed to an UIActivityIndicator now.

**Comment by kiliankoe** [comment]
Looks good :+1: Thanks! 😊

**Pull request merged**